### DISCUSSION TITLE
Is it possible to create 3D annotation from ortho views in Neuroglancer?

### DISCUSSION THREAD

**Discussion by nishantshakya** [google/neuroglancer#649] [Q&A]
The annotation mode only allows me to 2D boxes in the XY, YZ, and XZ views. I would like to create a bounding box in XY view and be able to edit it in other views to modify the size of Z. Is it feature available right now in Neuroglancer ?

**Comment by chourroutm** [comment]
Hi, yes! Once you've placed the first corner by Ctrl+Click, if you hover to another ortho view then you can place the second point in the other view and make it 3D.

I still don't find it very practical, thus I now create points only then use a short macro to extract their bounding box (if you have only two points, then you can edit the JSON state with their coordinates).
